.github/workflows/pr-checks.yml

```diff
@@ -78,7 +78,7 @@ jobs:
       - name: ktlint
         run: ./gradlew :stream-chat-android:ktlintCheck :stream-chat-android-sample:ktlintCheck
       - name: Unit tests
-        run: ./gradlew :stream-chat-android:test :stream-chat-android-sample:test --stacktrace
+        run: ./gradlew :stream-chat-android:testDebug :stream-chat-android-sample:testDebug --stacktrace
```

Comment:
We should remove the uploading of the release test results if we make this change, it's a couple lines down from here in a step mistakenly named `Upload testDebugUnitTest results` due to a copy-paste error 😄

Reply:
done

Reply:
This is there for all different jobs, not just the one, we should remove it from all of them.

Reply:
Oh, I see. Removed from all jobs now.